neo_pixel_state_machine.ino

```diff
@@ -1,75 +1,77 @@
-// -*- mode: c++ -*-
-// Neopixel state machine example
-//
-// Copyright (c) 2021 Dave Astels
-
-#include <Arduino.h>
-#include <Adafruit_NeoPixel.h>
-#include <Bounce2.h>
-#include "state_machine.h"
-#include "static_state.h"
-#include "breathe_1_state.h"
-#include "breathe_2_state.h"
-//#include "flash_1_state.h"
-//#include "flash_2_state.h"
-
-const uint32_t tick_interval = 5; // mS
-uint32_t tick_time = 0;
-
-Bounce mode_button = Bounce();
-Bounce red_button = Bounce();
-Bounce green_button = Bounce();
-Bounce blue_button = Bounce();
-Adafruit_NeoPixel strip = Adafruit_NeoPixel(1, 1, NEO_GRB + NEO_KHZ800);
```

Comment:
The pixel count and type should be extracted (using #define) and conditionally select  white support if the type is GRBW. I'll play with that.